ansible/roles/nfs-server/tasks/main.yml

```diff
@@ -31,9 +31,9 @@
     - name: Set ACLs on directories
       acl:
         path: "{{ item }}"
-        entity: nfsnobody
-        etype: user
-        permissions: rwx
+        entity: redteam
+        etype: group
+        permissions: rwX
```

Comment:
Seems to be an uppercase `X` instead of lowercase `x`

Reply:
`X` here will ensure any directories created by a user on the machine will belong to the `redteam` group and can be executed.